docs/process/Estimation-Process.md

```diff
@@ -129,6 +186,38 @@ main task and linked.
 We have found that Jira _subtasks_ aren't especially useful. They just
 make things confusing, so avoid using them.
 
+# Other methods of estimation
+
+## Story points estimates
```

Comment:
I think this could do with some clarification: Are we continuing to use both story points and "most probable"/"most pessimistic" estimates?

Reply:
It seems like story points estimates are being relegated to the "Other methods" section.
This would be quite a deviation from our current process. Much more of a change than just adding an additional field for "est. time remaining."
We would need to seek guidance from the holy men of scrum about this.

In any case it would be nice to add more detail to the story points section - e.g. transfer the info from Sergiy's presentation slide, give examples of 1,3,5 point tasks, etc.